config/sidekiq_scheduler.yml

```diff
@@ -287,12 +287,12 @@ VBADocuments::RunUnsuccessfulSubmissions:
   description: "Run VBADocuments::UploadProcessor for submissions that are stuck in uploaded status"
 
 VBADocuments::UploadRemover:
-  cron: "15,45 * * * * America/New_York"
+  cron: "*/2 * * * * America/New_York"
   class: VBADocuments::UploadRemover
   description: "Clean up submitted documents from S3"
 
 VBADocuments::UploadScanner:
-  cron: "0,30 * * * * America/New_York"
+  cron: "*/2 * * * * America/New_York"
```

Comment:
Similar to the job above, and every two minutes seemed reasonable enough to catch records that we failed to receive SNS notifications for. (This is a backup job for the `/upload_complete` endpoint.)